src/wr_logic_ai/nodes/README.md

```diff
@@ -0,0 +1,17 @@
+# Longrange AI
+
+@defgroup wr_logic_ai_longrange_ai Longrange Navigation
+@ingroup wr_logic_ai
+@brief Longrange Navigation Code
+
+This document describes the existing code and logics for long range navigation, which is primarily used in the autonomous navigation mission of URC. 
+
+## Long Range Navigation
+Long range navigation is primarily driven by a GPU sensor (which provides us the coordinate of the rover) and an IMU sensor (which provides us the heading of the rover). By listing the provided GNSS coordinates in the `coordinates.json` file, the rover would visit those locations in that specific order. The rover does so by finding the difference in coordinates and calculating a target heading using the difference.
```

Comment:
GPS?

Reply:
Good catch. Addressed in [4ac7f1f](https://github.com/WisconsinRobotics/WRover_Software/pull/85/commits/4ac7f1fbb2fb5f12141021c3cb3d3064e5c02f66).